app/assets/javascripts/admin/spree/orders/shipments.js.erb

```diff
@@ -0,0 +1,64 @@
+// Shipments AJAX API
+
+$(document).ready(function() {
+
+  handle_ship_click = function(){
+    var link = $(this);
+    var shipment_number = link.data('shipment-number');
+    var url = Spree.url( Spree.routes.orders_api + "/" + order_number + "/shipments/" + shipment_number + "/ship.json");
+    $.ajax({
+      type: "PUT",
+      url: url
+    }).done(function( msg ) {
+      window.location.reload();
+    }).error(function( msg ) {
+      console.log(msg);
+    });
+  }
+  // $('[data-hook=admin_order_edit_form] a.ship').click(handle_ship_click);
+
+  //handle shipping method edit click
```

Comment:
Are all these commented out lines in the originals files as well?

Reply:
yes, these events are registered in spree_backend. I will remove spree_backend and these comments in https://github.com/openfoodfoundation/openfoodnetwork/pull/4621/files